lib/defra_ruby/exporters/registration_export_service.rb

```diff
@@ -39,7 +65,7 @@ def write_to_file(report_class, file_path)
         Rails.logger.info "Finished writing #{file_path} from #{exporter_name}"
       end
 
-      def save_to_s3(export_type, file_path)
+      def self.save_to_s3(export_type, file_path)
         Rails.logger.info "Started daily upload of #{file_path} to s3"
```

Comment:
We just need to tweak the log message now this is being used for both exports, as the bulk ones won't be something that's done daily

```suggestion
        Rails.logger.info "Started upload of #{file_path} to s3"
```

Reply:
Fixed in 2cd2335.